find_composer_files.go

```diff
@@ -0,0 +1,19 @@
+package composer
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func FindComposerFiles(workingDir string) (composerJsonPath string, composerLockPath string, composerVar string, composerVarFound bool) {
```

Comment:
nit: can we get a comment about the purpose of this function?

Reply:
done